src/Angor/Client/Pages/Recover.razor

```diff
@@ -82,7 +87,8 @@
                     {
                         @if (StageInfo.CanRelease)
                         {
-                            <p>Total funds in penalty = @StageInfo.TotalInPenalty @network.CoinTicker</p>
+                            
+                            <p>Total funds in penalty = @(StageInfo.TotalInPenalty / 100_000_000) @network.CoinTicker</p>
```

Comment:
`TotalInPenalty` is a decimal and it should be a long it will then fix the issue
Same goes for `TotalSpendable`